tests/testthat/test-response.r

```diff
@@ -9,6 +9,29 @@ test_that("status codes returned as expected", {
 
 })
 
+test_that("DELETE deletes", {
+  expect_equal(GET("http://httpbin.org/delete")$status_code, 405)
+  expect_equal(DELETE("http://httpbin.org/delete")$status_code, 200)
+})
+
+test_that("POST posts", {
+  expect_equal(GET("http://httpbin.org/post")$status_code, 405)
+  expect_equal(POST("http://httpbin.org/post", body=list(),
```

Comment:
No. The POST, PUT, and PATCH tests also pass without body and encode. Would you prefer that I remove them?